includes/class-database-cleanup.php

```
@@ -267,15 +351,22 @@ public static function clean_expired_transients() {
+				LEFT JOIN $wpdb->posts p ON p.ID = pm.post_id
+				WHERE p.ID IS NULL LIMIT 5000"
+			);
 
-		return $result;
+			if ( $result ) {
+				$deleted += $result;
+			}
+		} while ( $result > 0 );
+
+		return $deleted;
 	}
```

PHPMD (2.15.0)

[error] The method clean_orphan_postmeta is not named in camelCase. (undefined)

(CamelCaseMethodName)